clang/test/SemaSYCL/accessors-targets-image.cpp

```diff
@@ -1,66 +1,83 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -ast-dump %s | FileCheck %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -ast-dump -sycl-std=2020 %s | FileCheck %s
 
-// This test checks that compiler generates correct kernel wrapper arguments for
+// This test checks if the compiler generates correct kernel wrapper arguments for
 // image accessors targets.
 
 #include "Inputs/sycl.hpp"
 
 using namespace cl::sycl;
```

Comment:
While you are working on this, perhaps a good opportunity to remove this line and use `sycl::` everywhere to make the code clearer for SYCL newbies?